dev-resources/Makefile.i18n

```diff
@@ -0,0 +1,123 @@
+# -*- Makefile -*-
+# This file was generated by the i18n leiningen plugin
+# It currently has a typo in autogeneration ">&" needs to be "&>"
+# Change your auto-generated makefile to look like this one 
+# There is a PR open at the moment, see for more details: https://github.com/puppetlabs/clj-i18n/pull/22 
```

Comment:
So would we normally have a .gitignore for the Makefile then?

Does the lein i18n plugin have a hook that causes that to get run automatically during builds?